### ISSUE TITLE
Fatal Error

### ISSUE THREAD

**Issue by blackhattristar** [PokeMapCommunity/PokeMap-iOS#6]
I'm getting this message at run time. fatal error: unexpectedly found nil while unwrapping an Optional value

**Comment by BrodyEller** [comment]
I'm getting the same thing. I'm trying to find the variable now.

**Comment by BrodyEller** [comment]
I haven't found the variable, but it must have been something to do with the watchlist being empty or nonexistent.  I installed the same PokeMap, with some tweaks, from [here](https://github.com/ivanbruel/PokeMap).  It is the same poke map, but it is missing some features.  Install it to your phone, add some pokemon onto the watchlist, then install this PokeMap on top of it and everything works fine.

**Comment by BrodyEller** [comment]
Scratch that, the error comes and goes.

**Comment by blackhattristar** [comment]
@BrodyEller Is there a fix without having to download the other PokeMap?

**Comment by BrodyEller** [comment]
I don't think that is a fix, because shortly after I got the error again.  It might have to do with Pokevision.com going down because the pokemon login servers went down.  Poke vision is what powers this and it was down all morning.

**Comment by blackhattristar** [comment]
@BrodyEller I noticed the site seemed to be down yesterday as well. Is there anywhere within the app to change the login credentials or access tokens because the map, information and access to the api still work but PokeVision is having problems. Maybe they got banned?

**Comment by BrodyEller** [comment]
I don't think they got banned.  The app is appending lat and long to the pokevision url and all the images are taken from pokevision.  I'm not sure if you can change the credentials though.  Is pokevision the only service that does this?  I've seen node.js api's on here, but not sure if they can find pokemon.

**Comment by BrodyEller** [comment]
And pokevision is down again.

**Comment by blackhattristar** [comment]
There is a better solution to PokeVision. [iPokeGO](https://github.com/istornz/iPokeGo) and [Pokemon Map](https://github.com/AHAAAAAAA/PokemonGo-Map) They are more difficult to setup but they work all the time and show more information than PokeVision.

**Comment by BrodyEller** [comment]
I want to try iPokeGo, but Xcode 8 is downloading rn.  Thats my one issue with it.

**Comment by reallyhp** [comment]
issue is with Pokevision API returning "id" and "pokemonId" as Int in JSON message
the model Pokemon.swift, has "id" and "pokemonId" as String
app crashes when trying to look up PokedexHelper.sharedInstance.nameFromId with nil
